desk/ted/lure-check-link.hoon

```diff
@@ -0,0 +1,33 @@
+/-  spider
+/+  *strandio
+=,  strand=strand:spider
+=,  strand-fail=strand-fail:libstrand:spider
+^-  thread:spider
+=/  m  (strand ,vase)
+|^  ted
+++  ted
+  |=  arg=vase
+  ^-  form:m
+  ;<  our=@p   bind:m  get-our
+  =/  arguments  !<((unit [cord ship cord path]) arg)
+  =/  [baseurl=cord target=ship group=cord =path]  (need arguments)
+  =/  target-tape  (trip (scot %p target))
+  ?~  target-tape  !!
```

Comment:
I'm pretty unfamiliar with threads. Is it ok to just crash? Should we maybe give output everywhere that we crash?

Reply:
Console output would be helpful here, I'll add that.

Reply:
It'd be very surprising for this to actually get hit though. The null check is just to mollify the type system so we can remove the initial sig.